src/test/resources/pages/instructorFeedbackResultsSortGiverRecipientQuestionTeam.html

```diff
@@ -1939,7 +1939,7 @@
                                   [more]
                                 </a>
                                 <br>
-                                <span id="questionAdditionalInfo-8-section-1-giver-1-recipient-1" style="display: none;">
+                                <span id="questionAdditionalInfo-8-section-1-giver-1-recipient-1" style="display:none;">
```

Comment:
@chowyb Note that in this commit dded058 I actually swap the checking and I run god mode to change the `html` file or the test case will fail.
The missing space is actually widely existing in the whole project as `feedbackQuestionAdditionalInfoTemplate.html` and `feedbackQuestionContribAdditionalInfoTemplate.html` don't follow the convention
``` html
&nbsp;<span style=" white-space: normal;">
    <a href="javascript:;" id="questionAdditionalInfoButton-${questionNumber}-${additionalInfoId}" class="color_gray" onclick="toggleAdditionalQuestionInfo('${questionNumber}-${additionalInfoId}')" data-more="${more}" data-less="${less}">${more}</a>
    <br>
    <span id="questionAdditionalInfo-${questionNumber}-${additionalInfoId}" style="display:none;">${questionAdditionalInfo}</span>
</span>
```
``` html
&nbsp;<span style=" white-space: normal;">
    <a href="javascript:;" id="questionAdditionalInfoButton-${questionNumber}-${additionalInfoId}" class="small link-in-dark-bg" onclick="toggleAdditionalQuestionInfo('${questionNumber}-${additionalInfoId}')" data-more="${more}" data-less="${less}">${more}</a>
    <br>
    <span id="questionAdditionalInfo-${questionNumber}-${additionalInfoId}" style="display:none;">${questionAdditionalInfo}</span>
</span>
```
When JS modifying the DOM, it actually **add** a whitespace (this is why when the comparison is done **after** the checking of toggling, it got whitespace)